protoc-gen-gogo/generator/helper.go

```diff
@@ -181,10 +181,19 @@ func (g *Generator) GetFieldName(message *Descriptor, field *descriptor.FieldDes
 			return fieldname + "_"
 		}
 	}
+	if gogoproto.IsProtoSizer(file.FileDescriptorProto, message.DescriptorProto) {
```

Comment:
But I can't add both Size and ProtoSize to reserved names - it will try to put suffix on them always. Or I missed something?